app/views/layouts/application.html.erb

```diff
@@ -27,6 +27,9 @@
           <li><%= link_to 'Sign In', new_user_session_path %></li>
         <%- end -%>
       </ul>
+      <ul class = "left">
+        <li><%= button_to "New Wizard", new_wizard_path,method: :get %></li>
```

Comment:
generally I would recommend the use of an anchor tag here as opposed to a button. Semantically we are "linking" to a page that has a wizard form on it.